nsis/src/nsis/NsisLoader.java

```diff
@@ -156,50 +150,60 @@ private void initScriptHeader(FileBytes fileBytes,
 	 * @param dataType to apply to the bytes
 	 * @return
 	 */
-	private Data createData(Program program, Listing listing, Address address,
-			DataType dt) {
+	private Data createData(Program program, Address address, DataType dt) {
 		try {
+			Listing listing = program.getListing();
 			Data d = listing.getDataAt(address);
 			if (d == null || !dt.isEquivalent(d.getDataType())) {
 				d = DataUtilities.createData(program, address, dt, -1, false,
 						ClearDataMode.CLEAR_ALL_UNDEFINED_CONFLICT_DATA);
 			}
 			return d;
 		} catch (Exception e) {
-			e.printStackTrace();
+			e.printStackTrace(); // TODO use Ghidra logging system to manage exceptions
```

Comment:
nit: We probably just want to the allow the exception to be thrown here, if we can't create the data using DataUtilities, I suspect all is lost.

Reply:
Fixed, I agree that the exception should be thrown